src/components/application_manager/src/application_manager_impl.cc

```diff
@@ -1404,16 +1404,16 @@ void ApplicationManagerImpl::SendMessageToMobile(
                                  << ") not allowed by policy");
       return;
     }
-
-    if (function_id == mobile_apis::FunctionID::OnSystemRequestID) {
+    // Moving OnUpdateRequestSentToMobile OnSnapshotCreated method.
+   /* if (function_id == mobile_apis::FunctionID::OnSystemRequestID) {
       mobile_apis::RequestType::eType request_type =
           static_cast<mobile_apis::RequestType::eType>(
               (*message)[strings::msg_params][strings::request_type].asUInt());
       if (mobile_apis::RequestType::PROPRIETARY == request_type ||
           mobile_apis::RequestType::HTTP == request_type) {
         GetPolicyHandler().OnUpdateRequestSentToMobile();
       }
-    }
+    }*/
```

Comment:
@madhuy One small issue with this, the Travis build uses a style checking script (located [here](https://github.com/smartdevicelink/sdl_core/blob/master/tools/infrastructure/check_style.sh)) and is currently failing on this block, saying that the block should be moved forward by one space. 

```
$ bash -e tools/infrastructure/check_style.sh

<    /* if (function_id == mobile_apis::FunctionID::OnSystemRequestID) {
<       mobile_apis::RequestType::eType request_type =
<           static_cast<mobile_apis::RequestType::eType>(
<               (*message)[strings::msg_params][strings::request_type].asUInt());
<       if (mobile_apis::RequestType::PROPRIETARY == request_type ||
<           mobile_apis::RequestType::HTTP == request_type) {
<         GetPolicyHandler().OnUpdateRequestSentToMobile();
<       }
<     }*/
---
>     /* if (function_id == mobile_apis::FunctionID::OnSystemRequestID) {
>        mobile_apis::RequestType::eType request_type =
>            static_cast<mobile_apis::RequestType::eType>(
>                (*message)[strings::msg_params][strings::request_type].asUInt());
>        if (mobile_apis::RequestType::PROPRIETARY == request_type ||
>            mobile_apis::RequestType::HTTP == request_type) {
>          GetPolicyHandler().OnUpdateRequestSentToMobile();
>        }
>      }*/

The command "bash -e tools/infrastructure/check_style.sh" exited with 1.
```

You can run this script on your local machine from your sdl_core folder by running this command:

```bash
bash -e tools/infrastructure/check_style.sh
```

You may need to install `clang-format-3.6` beforehand using `sudo apt-get install clang-format-3.6`